index.md

```diff
@@ -0,0 +1,28 @@
+---
+---
+
+Wobscale is a Seattle-based creative technology collective. We poke fun at terms like *web scale* and *the cloud* while also swearing by them in the same breath.
```

Comment:
I feel like the second sentence here needs rephrasing, but it's been 10 days and I haven't thought of a better way to say it...

Reply:
I kinda like it, but I'll throw a few sentences at the wall and see if any stick better.

"We defy simple explanation, existing as both a parody of modern tech while simultaneously embodying it."

"We exist partially to poke fun at the industry, but also as a front for an eclectic collection of projects. We explicitly eschew making money in favor of having fun and working on things we care about."

"What is a Wobscale you may ask? Wobscale, a corruption of "web scale", is a front for a loose collective of individuals and projects. We strive to both create cool things, have fun, and make friends along the way (and those friends were the real Wob all along)."

Reply:
I think I want to take @euank's first suggestion to replace the sentence in question, then add another paragraph with the second suggestion.